core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java

```diff
@@ -51,7 +51,7 @@
     private DataSize smallPartitionedMaxSizePerOperator = DataSize.of(500, KILOBYTE);
     private DataSize smallMaxSizePerFilter = DataSize.of(5, MEGABYTE);
 
-    private int largeMaxDistinctValuesPerDriver = 10_000;
+    private int largeMaxDistinctValuesPerDriver = 100_000;
```

Comment:
Given that this is a substantial increase, is this increase benchmarked separately from the "small" one ? IIUC we're only benchmarking "small" DF limits in our usual perf benchmarks

Reply:
Could you please elaborate why do we see performance regression with increasing the limits? The values are collected with a hash set, and it's performance shouldn't vary dramatically with an increase of it's size. The complexity is still O(1) assuming right fill ration, maybe only cache misses may increase (though the total memory footprint is limited?).